foundations/the_back_end/introduction_to_the_backend_lesson.md

```diff
@@ -27,6 +30,6 @@ As we covered back in the [Installations section](https://www.theodinproject.com
 This section contains questions for you to check your understanding of this lesson. If you're having trouble answering the questions below on your own, review the material above to find the answer.
 
 * What is back-end development?
-* How many parts does the back-end consist of?
-* Name each part of the back-end.
-* Where does the back-end store its data?
+* How many parts does the back end consist of?
```

Comment:
In the learning outcomes, we use the hyphen, so it's probably best to maintain that here.

Reply:
But "the back end" is a noun phrase, whereas "back-end", as found elsewhere, is a compound adjective.

Reply:
I'm a proponent of either leaving it hyphenated or changing it to "backend" in all places.

I think colloquialism in the context of this particular domain should trump what is considered to be "proper".

Reply:
Ok, but I don't think "back end" in any form is colloquial...? In another lesson, there's a clause that says "patch 'er up", which I would say is colloquial.

I think it's about consistency above all, and also clarity, and I think that a distinction between the noun and adjective enhances that.